pyiron_ontology/parser.py

```diff
@@ -1,7 +1,8 @@
 from semantikon.converter import parse_input_args, parse_output_args
-from semantikon.ontology import get_knowledge_graph, PNS
+from semantikon.ontology import get_knowledge_graph, SNS
 from rdflib import Graph
-from pyiron_workflow import NOT_DATA, Workflow, Macro
+from pyiron_workflow import Workflow
```

Comment:
If you want you can lump this into the api import; the top init pulls everything from the api anyhow. I did this exactly because I'm too lazy/aesthetically inclined to not have the imports split over two lines 😂 if you like it better this way, I'm fine with it as is

Reply:
ah true ok